internal/pkg/cli/validate.go

```diff
@@ -626,6 +636,69 @@ func validateLSIs(val interface{}) error {
 	return nil
 }
 
+func validateSubscribe(noSubscription bool, subscribeTags []string) error {
+	// --no-subscriptions and --subscribe are mutually exclusive.
+	if noSubscription && len(subscribeTags) != 0 {
+		return fmt.Errorf("validate subscribe configuration: cannot specify --%s and --%s options at once", noSubscriptionFlag, subscribeFlag)
```

Comment:
nit: to be consistent with `task run` errors
```suggestion
		return fmt.Errorf("validate subscribe configuration: cannot specify both --%s and --%s", noSubscriptionFlag, subscribeFlag)
```